src/components/Address/Address.js

```diff
@@ -250,14 +252,18 @@ export default function Address({
             setActiveSort={setActiveSort}
             url={url}
           />
-          <Sorting
-            activeSort={activeSort}
-            setActiveSort={setActiveSort}
-            activeFilter={domainType}
-          />
 
-          {domainType === 'registrant' && (
-            <>
+          <div
```

Comment:
Please refrain from inlining css styling. Please follow [this guide ](https://github.com/ensdomains/ens-app#cssstyling) for more detail.